sysdig/internal/client/v2/ibm_test.go

```diff
@@ -168,7 +168,6 @@ func TestIBMClient_CurrentTeamID(t *testing.T) {
 		if err != nil {
 			t.Errorf("failed to create response, err: %v", err)
 		}
-		return
```

Comment:
This is out of the scope of this PR

Reply:
Its not, actually this is changed due to linting stuff. I had to change it.

Reply:
Why its not failing before? Because linter was ignoring `*_test.go` files with tags.